notebooks/Tutorial_templates/tutorial_template_msb_v1.ipynb

```diff
@@ -0,0 +1,139 @@
+{
```

Comment:
<p>Generally we have sub headings categorized by action or step in a given workflow. We may want to clarify here that several sub headings should exist with each step (i.e. 3.2 Discover Data, 3.3. Access Data, etc.)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/nsidc/NSIDC-Data-Tutorials/pull/36/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>